README.md

```diff
@@ -258,6 +259,29 @@ manage and re-use contexts.
 - `validate` allows for an initial validation of the context and any provided configuration. If the context and configuration are OK to run the job, returning `spark.jobserver.SparkJobValid` will let the job execute, otherwise returning `spark.jobserver.SparkJobInvalid(reason)` prevents the job from running and provides means to convey the reason of failure. In this case, the call immediately returns an `HTTP/1.1 400 Bad Request` status code.  
 `validate` helps you preventing running jobs that will eventually fail due to missing or wrong configuration and save both time and resources.  
 
+### NEW SparkJob API
+
+Note: As of version 0.7.0, a new SparkJob API that is significantly better than the old SparkJob API will take over.  Existing jobs should continue to compile against the old `spark.jobserver.SparkJob` API, but this will be deprecated in the future.  Note that jobs before 0.7.0 will need to be recompiled, older jobs may not work with the current SJS example.  The new API looks like this:
```

Comment:
Good Q.  Well that’s why it will be 0.7, and not 0.6.3, but maybe you’re right that it’s a big enough change for 1.0.   Well, at the very least, if we extend the SparkJobFactory to support Python jobs, that will definitely be worth a 1.0. 

My only thoughts is that we should make a 1.0 more stable and free of bugs than today, and we need to think about Spark versioning.  Spark 2 will be out soon, and some will clamor for that, but lots of folks (most of us I imagine) will have systems on Spark 1.x for a while in production, and not ready to move up.

> On Jun 1, 2016, at 10:17 AM, Fernando Otero <notifications@github.com> wrote:
> 
> In README.md <https://github.com/spark-jobserver/spark-jobserver/pull/442#discussion_r65403989>:
> 
> > @@ -258,6 +259,29 @@ manage and re-use contexts.
> >  - `validate` allows for an initial validation of the context and any provided configuration. If the context and configuration are OK to run the job, returning `spark.jobserver.SparkJobValid` will let the job execute, otherwise returning `spark.jobserver.SparkJobInvalid(reason)` prevents the job from running and provides means to convey the reason of failure. In this case, the call immediately returns an `HTTP/1.1 400 Bad Request` status code.  
> >  `validate` helps you preventing running jobs that will eventually fail due to missing or wrong configuration and save both time and resources.  
> >  
> > +### NEW SparkJob API
> > +
> > +Note: As of version 0.7.0, a new SparkJob API that is significantly better than the old SparkJob API will take over.  Existing jobs should continue to compile against the old `spark.jobserver.SparkJob` API, but this will be deprecated in the future.  Note that jobs before 0.7.0 will need to be recompiled, older jobs may not work with the current SJS example.  The new API looks like this:
> wouldn't a change like this deserve a new major version? 1.0?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/spark-jobserver/spark-jobserver/pull/442/files/6c9e4d963c90a8f9ce50b818440d3f2c16149699#r65403989>, or mute the thread <https://github.com/notifications/unsubscribe/ABA32_PrbQVelO_-hKacFdBvaZZdMilDks5qHb6lgaJpZM4IQq8l>.
>